### PR TITLE
Feature/implement metrics module

### PR DESCRIPTION
**Pull request by santiagonoguez**
Please review the collecting part of the card.

### CODE REVIEW

File: src/collecting/disk.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+// currently common-display-module depends on rise-common-electron, this may change in the future to avoid electron dependencies.
+const {platform} = require("rise-common-electron")
```
Comment (rodrigopavezi) [inline src/collecting/disk.js]: @santiagonoguez could you please also do the work of moving the getFreeDiskSpace() function from rise-common-electron to common-display-module? cheers

Comment (santiagonoguez) [reply src/collecting/disk.js]: in rise-common-electron this function is part of a platform.js file, but common-display-module has all it's definitions in a single common.js file, should I keep the function in a separated platform.js file or should I add it to the common.js file as the rest of the functions are ?

Comment (rodrigopavezi) [reply src/collecting/disk.js]: @santiagonoguez I think the best is keeping on a separated file and exports it through the common.js. cheers

**Comment by rodrigopavezi** [comment]
@santiagonoguez before merging lets setup circle ci to build and delivery it.

**Pull request closed without merging**